File: csharp/1020.cs
using System; 

class URI {

    static void Main(string[] args) { 

        int entrada, ano, mes, dia;
        int.TryParse(Console.ReadLine(), out entrada);
        ano = entrada/365;
        mes = entrada%365/30;
        dia = entrada%365%30;
        Console.WriteLine($"{ano} ano(s)");
        Console.WriteLine($"{mes} mes(es)");
        Console.WriteLine($"{dia} dia(s)");
    }

}

File: java 19/1020.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner sc = new Scanner(System.in);
        int ano = 0, mes = 0, dia = 0;
        int entrada = sc.nextInt();
        sc.nextLine();
        ano = entrada / 365;
        mes = entrada % 365 / 30;
        dia = entrada % 365 % 30;
        System.out.printf("%d ano(s)\n",ano);
        System.out.printf("%d mes(es)\n",mes);
        System.out.printf("%d dia(s)\n",dia);
        sc.close();
 
    }
 
}
